src/Pager/PagerInterface.php

```diff
@@ -147,31 +147,23 @@ public function setCursor(int $pos): void;
      * Returns an object by cursor position.
      *
      * @param int $pos
```

Comment:
this can be dropped to

Reply:
I noticed but there are a lot of similar issues, this should be fixed on a separate PR targetting the stable branch IMO